packages/utilities/gel-foundations-styled-components/src/typography.js

```diff
@@ -1,63 +1,196 @@
   }
 `;
 
-export const GEL_BREVIER = css`
-  font-size: 0.875em;
-  line-height: 1rem;
+export const GEL_IMPERIAL = css`
+  font-size: 3.125em;
+  line-height: 3.375rem;
+
   ${MEDIA_QUERY_TYPOGRAPHY.SMART_PHONE_ONLY} {
-    line-height: 1.125rem;
+    font-size: 4em;
+    line-height: 4.5rem;
+  }
+
+  ${MEDIA_QUERY_TYPOGRAPHY.SMART_PHONE_ONLY} {
+    font-size: 6em;
+    line-height: 6.5rem;
   }
+`;
+
+export const GEL_ROYAL = css`
+  font-size: 2.5em;
+  line-height: 2.75rem;
+
+  ${MEDIA_QUERY_TYPOGRAPHY.SMART_PHONE_ONLY} {
+    font-size: 3.25em;
+    line-height: 3.75rem;
+  }
+
   ${MEDIA_QUERY_TYPOGRAPHY.LAPTOP_AND_LARGER} {
-    font-size: 0.8125em;
+    font-size: 4.75em;
+    line-height: 5.25rem;
+  }
+`;
+
+export const GEL_FOOLSCAP = css`
+  font-size: 2em;
+  line-height: 2.25rem;
+
+  ${MEDIA_QUERY_TYPOGRAPHY.SMART_PHONE_ONLY} {
+    font-size: 2.5em;
+    line-height: 2.75rem;
+  }
+
+  ${MEDIA_QUERY_TYPOGRAPHY.LAPTOP_AND_LARGER} {
+    font-size: 3.5em;
+    line-height: 3.75rem;
   }
 `;
 
-/* Font styling below is a subset of BBC GEL Typography "Canon" */
 export const GEL_CANON = css`
   font-size: 1.75em;
   line-height: 2rem;
+
   ${MEDIA_QUERY_TYPOGRAPHY.SMART_PHONE_ONLY} {
     font-size: 2em;
     line-height: 2.25rem;
   }
+
   ${MEDIA_QUERY_TYPOGRAPHY.LAPTOP_AND_LARGER} {
     font-size: 2.75em;
     line-height: 3rem;
   }
 `;
 
-/* Font styling below is a subset of BBC GEL Typography "Trafalgar" */
 export const GEL_TRAFALGAR = css`
   font-size: 1.25em;
   line-height: 1.5rem;
+
   ${MEDIA_QUERY_TYPOGRAPHY.SMART_PHONE_ONLY} {
     font-size: 1.5em;
     line-height: 1.75rem;
   }
+
   ${MEDIA_QUERY_TYPOGRAPHY.LAPTOP_AND_LARGER} {
     font-size: 2em;
     line-height: 2.25rem;
   }
 `;
 
+export const GEL_PARAGON = css`
+  font-size: 1.25em;
+  line-height: 1.5rem;
+
+  ${MEDIA_QUERY_TYPOGRAPHY.SMART_PHONE_ONLY} {
+    font-size: 1.375em;
+    line-height: 1.625rem;
+  }
+
+  ${MEDIA_QUERY_TYPOGRAPHY.LAPTOP_AND_LARGER} {
+    font-size: 1.75em;
+    line-height: 2rem;
+  }
+`;
+
+export const GEL_DOUBLE_PICA = css`
+  font-size: 1.25em;
+  line-height: 1.5rem;
+
+  ${MEDIA_QUERY_TYPOGRAPHY.SMART_PHONE_ONLY} {
+    line-height: 1.5rem;
+  }
+
+  ${MEDIA_QUERY_TYPOGRAPHY.LAPTOP_AND_LARGER} {
+    font-size: 1.5em;
+    line-height: 1.75rem;
+  }
+`;
+
+export const GEL_GREAT_PRIMER = css`
+  font-size: 1.125em;
+  line-height: 1.375rem;
+
+  ${MEDIA_QUERY_TYPOGRAPHY.SMART_PHONE_ONLY} {
+    line-height: 1.375rem;
+  }
+
+  ${MEDIA_QUERY_TYPOGRAPHY.LAPTOP_AND_LARGER} {
+    font-size: 1.25em;
+    line-height: 1.5rem;
+  }
+`;
+
+export const GEL_BODY_COPY = css`
+  font-size: 0.9375em;
+  line-height: 1.25rem;
+
+  ${MEDIA_QUERY_TYPOGRAPHY.SMART_PHONE_AND_LARGER} {
+    font-size: 1em;
+    line-height: 1.375rem;
+  }
+`;
+
+export const GEL_PICA = css`
+  font-size: 0.9375em;
+  line-height: 1.25rem;
+
+  ${MEDIA_QUERY_TYPOGRAPHY.SMART_PHONE_AND_LARGER} {
+    font-size: 1em;
+    line-height: 1.25rem;
```

Comment:
Line height doesn't change but font size does?

Reply:
Yep -- definitely a bit weird, it can be verified (& looks slightly more reasonable) with the [GEL table](https://www.bbc.co.uk/gel/guidelines/typography#type-sizes).